Cargo.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 
 [dependencies]
 winrt_macros = { path = "crates/macros",  version = "0.7.1" }
-sha1 = "0.6.0"
+const-sha1 = { path = "../const-sha1" }
```

Comment:
Where is this crate going to live?

Reply:
It's on my personal GitHub and already live on crates.io: https://github.com/rylev/const-sha1 https://crates.io/search?q=const-sha1

Reply:
Got it - so we just need that update and then we can point to the crate.

Reply:
Exactly. I just didn't want to push a new version if we decided in this PR that the API wasn't quite right